internal/common/metrics/service_wrapper.go

```diff
@@ -117,10 +119,14 @@ func (w *workflowServiceMetricsWrapper) getOperationScope(scopeName string) *ope
 func (s *operationScope) handleError(err error) {
 	s.scope.Timer(CadenceLatency).Record(time.Since(s.startTime))
 	if err != nil {
-		st := status.Convert(err)
-		if st.Code() == codes.NotFound || st.Code() == codes.InvalidArgument || st.Code() == codes.AlreadyExists {
+		switch err.(type) {
```

Comment:
This is how it looked like in original version.

Reply:
Should we use errors.As(...)? 
https://blog.golang.org/go1.13-errors

Reply:
I did consider it. I even have this link still opened. Honestly I am in doubts. `errors.As()` requires one more variable declaration and in most cases (like this one) we don't actually need error itself. Also I don't want to mix patterns. If we decide to switch to `errors.As()` we need to do it everywhere, I believe.